application/namespaces/__init__.py

```diff
@@ -0,0 +1,2 @@
+from .notifications import notification_ns
+from .main import ns as main_ns
```

Comment:
@Edison-Stuart Why `ns` instead of `main_ns`? We could just define the main namespace as `main_ns` in `application.namespaces.main`.